### DISCUSSION TITLE
How do I allow UPnP?

### DISCUSSION THREAD

**Discussion by tsuza** [firewalld/firewalld#1392] [Q&A]
Firewalld keeps blocking UPnP-related connections, even though my device is allowed to port forward through UPnP and the port has been port forwarded. If I disable firewalld, it works just fine.

I've tried adding upnp-client as a service, allowing the usual upnp ports, adding a rich rule to multicast connections on the 224.0.0.0 ip, and so on and so forth, yet nothing.

What else am I missing?

**Comment by unknowndevQwQ** [comment]
Please provide more information, such as upnp server configuration file, system version, firewalld version.
